test/FT-voting-quadratic/RIFGovernorFtQuadratic.test.js

```diff
@@ -171,21 +183,64 @@ describe('Governance - Fungible tokens voting', () => {
             reason,
           );
       });
+    });
```

Comment:
Perhaps after this add another test for 
`it('voters should have finished voting', async () => {`
and make it similar to 
`it('voters should not have voted yet', async () => {`